### PR TITLE
Feature borrar mis alertas 159958713

### PR DESCRIPTION
**Pull request by FedePe**
This PR implements the removal of one of my alerts.
Clicking on the image of the trash can sends the id of the alert that you want to delete to the backend.

### CODE REVIEW

File: src/components/wyehome/MyAlert.jsx [hunk 1]
```diff
@@ -1,28 +1,73 @@
...
+  onDismiss() {
+    const { id } = this.props;
+    fetch(`${process.env.REACT_APP_API_URL}/alerts/${id}`, {
+      method: 'DELETE',
+      headers: {
+        Authorization: `Bearer ${window.localStorage.getItem('sessionToken')}`,
+      },
+    }).then((response) => {
+      if (!response.ok) {
+        throw Error('');
```
Comment (elcvilla) [inline src/components/wyehome/MyAlert.jsx]: We should send a message in case of error. Do you agree?

File: src/components/wyehome/MyAlert.jsx [hunk 2]
```diff
@@ -1,28 +1,73 @@
...
+      }
+    })
+      .then(() => { this.setState({ visible: false }); })
+      .catch(() => {
```
Comment (elcvilla) [inline src/components/wyehome/MyAlert.jsx]: We should send a message in case of error. Do you agree?

File: src/components/wyehome/MyAlert.jsx [hunk 3]
```diff
@@ -1,28 +1,73 @@
...
+  render() {
+    const {
+      description, type,
+    } = this.props;
+    const {
+      visible,
+    } = this.state;
+
+    return (
+
```
Comment (rocespedes) [inline src/components/wyehome/MyAlert.jsx]: nit: unnessary extra line

File: src/components/wyehome/MyAlert.jsx (hunk 2 above)
Comment (FedePe) [reply src/components/wyehome/MyAlert.jsx]: You are right! Done

File: src/components/wyehome/MyAlert.jsx (hunk 1 above)
Comment (FedePe) [reply src/components/wyehome/MyAlert.jsx]: You are right! Done

File: src/components/wyehome/MyAlert.jsx (hunk 3 above)
Comment (FedePe) [reply src/components/wyehome/MyAlert.jsx]: Done!

**Pull request merged**